res/css/_common.scss

```diff
@@ -250,6 +250,7 @@ textarea {
 .mx_Dialog button.danger, .mx_Dialog input[type="submit"].danger {
     background-color: $warning-color;
     border: solid 1px $warning-color;
+    color: $accent-fg-color;
```

Comment:
From a code perspective, fine. From a UI perspective though, this text should very much be white.

Reply:
Currently the text on all the red buttons is black in the dark theme

Reply:
Oh. But isn't accent-fg-color green?

Reply:
Oh. Sorry, I should've looked at what the colours actually are.